server/models/data/availabilitydetails.js

```diff
@@ -0,0 +1,19 @@
+const moment = require('moment');
+
+class AvailabilityDetails {
+
+    constructor(availabilityData) {
+        this.schoolId = availabilityData.schoolId_id;
```

Comment:
Hi Nancy,
I would recommend revising this line to `school_id` as the database does not have a value for `schoolId_id`.

Reply:
Hi Amy,
Thank you for catching that!
My action: I've changed the name from `schoolId_id` to `school_id`. 
You can now see these changes in line 6 of server/models/db>student.js
line 18 of server/models/data/availabilitydetails.js.

Please confirm if this takes care of your suggestion!